src/shared/compounders/AMMCompounderBase.sol

```diff
@@ -36,6 +36,8 @@ abstract contract AMMCompounderBase is ReentrancyGuard, ERC4626 {
     using FixedPointMathLib for uint256;
     using SafeERC20 for IERC20;
 
+    /// @notice The internal accounting of the deposit limit. Denominated in shares.
+    uint256 public depositCap;
```

Comment:
If the cap is denominated in vault shares, does that mean we have to do math in order to set a certain Dollar limit?

Reply:
All of the math is done by the contract, and is callable from on-chain without any restrictions.

The cap is denominated in shares because `harvest`s increase the `totalAUM`.

We wouldn't be able to automatically set a Dollar limit in any case, as the assets are changing in value + we probably wouldn't want to use a price oracle for that.

Reply:
Ok so I am reading the answer to my question is "yes". 😄